run/GCClassic/geoschem_config.yml.templates/geoschem_config.yml.CH4

```diff
@@ -83,6 +83,8 @@ CH4_simulation_options:
     state_vector_element_number: 0
     use_emission_scale_factor: false
     use_OH_scale_factors: false
+    perturb_CH4_boundary_conditions: false
+    CH4_boundary_condition_ppb_increase_NSEW: [0.0, 0.0, 0.0, 0.0]
```

Comment:
This line should also be added to the geoschem_config.yml files for tagCH4 and carbon simulations.